tests/automation_framework/data/worker/workerregister_workOrderPayloadFormats_JSON_RPC_JWT.json

```diff
@@ -0,0 +1,26 @@
+{
+    "jsonrpc": "2.0",
+    "method": "WorkerRegister",
+    "id": 1,
+    "params": {
+        "workerId":"",
+        "workerType": 1,
+        "organizationId":"0xAD",
+        "applicationTypeId": "0XAD",
+        "workerEncryptionKey" : " " ,
+        "details": {
+            "hashingAlgorithm": "SHA-256",
+            "signingAlgorithm": "SECP256K1" ,
+            "keyEncryptionAlgorithm": "RSA-OAEP-3072",
+            "dataEncryptionAlgorithm": "AES-GCM-256",
+	    "workOrderPayloadFormats": ["JSON-RPC-JWT"],
```

Comment:
"workOrderPayloadFormats" is not aligned in line with the other parameters above.

Reply:
Do you also want to add a test for workOrderPayloadFormats: ["JSON-RPC"]